venues/OpenReview.net/Support/deployProcess.py

```diff
@@ -5,7 +5,7 @@ def process(client, note, invitation):
     conference = openreview.helpers.get_conference(client, note.forum)
     print(conference.get_id())
     forum = client.get_note(id=note.forum)
-    comment_readers = forum.content['Contact Emails']
+    comment_readers = forum.content['Contact Emails'][:]
```

Comment:
Isn't bettter to this something like this:

```
comment_readers = forum.content['Contact Emails'] + ['OpenReview.net/Support']
```

Reply:
Agreed. Changes have been made in accordance.